.github/workflows/README.md

```diff
@@ -0,0 +1,105 @@
+# Workflow Taxonomy
+
+## Where and what tests are run
+
+---
+
+### Workflow: `deploy-happy-stack.yml`
+
+#### Ran on:
+
+- `deployment`: (deployments are triggered by `push-tests`)
+
+#### Jobs:
+
+- ##### `upgrade`: TODO
```

Comment:
Mind opening an issue to fill out these remaining job steps and link it in the Improve Testing Confidence Documentation epic?